fdbclient/Schemas.cpp

```diff
@@ -501,7 +501,8 @@ const KeyRef JSONSchemas::statusSchema = LiteralStringRef(R"statusSchema(
              "ssd-1",
              "ssd-2",
              "ssd-redwood-experimental",
-             "memory"
+             "memory",
+             "old_memory"
```

Comment:
Should we use a model like we did with ssd that uses a version? Not only is "old" inconsistent, but we can only use it once.

Also, even if we kept it, we should probably use a hyphen rather than an underscore for consistency.